### ISSUE TITLE
Prettier preflight option causes warnings

### ISSUE THREAD

**Issue by surchs** [neurobagel/query-tool#81]
We have this line in our prettier config that was added during the tailwind install
https://github.com/neurobagel/query-tool/blame/99fc5c019054aff221280b37ab1a09886721134a/.prettierrc.json#L9-L11

When I run prettier locally, I get a bunch of:

```
[warn] Ignored unknown option { corePlugins: { preflight: false } }
```
Warnings. They don't do anything, but they look ugly - and this could be prettier :drum: 

I also can't find this `corePlugins` section referenced anywhere in the docs for tailwind or the tailwind prettier plugin: https://github.com/tailwindlabs/prettier-plugin-tailwindcss?tab=readme-ov-file#installation

So I think we could just remove this

**Comment by rmanaem** [comment]
> I also can't find this corePlugins section referenced anywhere in the docs for tailwind or the tailwind prettier plugin: https://github.com/tailwindlabs/prettier-plugin-tailwindcss?tab=readme-ov-file#installation

This is from the mui docs, see https://mui.com/material-ui/integrations/interoperability/#tailwind-css  and we may have copy pasted this from the tailwind config during setup

> So I think we could just remove this

I believe we can remove it from `prettierrc.json`

**Comment by surchs** [comment]
:rocket: Issue was released in [`v0.2.0`](https://github.com/neurobagel/query-tool/releases/tag/v0.2.0) :rocket:
